packages/core/src/browser/test/jsdom.ts

```diff
@@ -7,26 +7,42 @@
 
 import { JSDOM } from 'jsdom';
 
-const dom = new JSDOM('<!doctype html><html><body></body></html>');
-
-(global as any)['document'] = dom.window.document;
-(global as any)['window'] = dom.window;
-(global as any)['navigator'] = { userAgent: 'node.js', platform: 'Mac' };
-(global as any)['HTMLElement'] = (global as any)['window'].HTMLElement;
-
-const toCleanup: string[] = [];
-Object.getOwnPropertyNames((dom.window as any)).forEach(property => {
-    if (typeof (global as any)[property] === 'undefined') {
-        (global as any)[property] = (dom.window as any)[property];
-        toCleanup.push(property);
+/**
+ * ```typescript
+ * const disableJSDOM = enableJSDOM();
+ * // actions require DOM
+ * disableJSDOM();
+ * ```
+ */
+export function enableJSDOM(): () => void {
+    if (typeof (global as any)['_disableJSDOM'] === 'function') {
+        return (global as any)['_disableJSDOM'];
     }
-});
+    const dom = new JSDOM('<!doctype html><html><body></body></html>');
 
-(dom.window.document as any)['queryCommandSupported'] = function () { };
+    (global as any)['document'] = dom.window.document;
+    (global as any)['window'] = dom.window;
+    (global as any)['navigator'] = { userAgent: 'node.js', platform: 'Mac' };
```

Comment:
Could you add a comment why `Mac`?

Reply:
`platform` just needs to be defined for some module to be loaded.